src/de.rs

```diff
@@ -316,6 +316,18 @@ impl<'de, R: Read<'de>> Deserializer<R> {
             }
         };
 
+        // to deserialize floats we'll first push the 'integer' and 'fraction' parts, both as byte
```

Comment:
this comment explains how `minimal-lexical` has been integrated into the existing code base